flutter/lib/models/file_model.dart

```diff
@@ -496,6 +501,35 @@ class FileController {
       debugPrint(
           "path: ${from.path}, toPath: $toPath, to: ${PathUtil.join(toPath, from.name, isWindows)}");
     }
+
+    final List<Entry> entrys = items.items.toList();
+
+    await Future.forEach(entrys, (Entry item) async {
+      final List<String> paths = [];
+
+      final emptyDirs =
+          await fileFetcher.readEmptyDirs(item.path, isLocal, showHidden);
+
+      if (emptyDirs.isEmpty) {
+        paths.add(item.path);
```

Comment:
1. If `emptyDirs` is empty, dothing, otherwise there will be a empty directory whose name is the same as the current item path. After changing this, tests during win<->win, win<->linux, win<->andrioid are good.
2. Add version check:
```
if (versionCmp(rootState.target!.ffiModel.pi.version, '1.3.3') < 0) {
      return;
    }
```
3. Remove changes in `pubspec.lock` and code format in `message.proto`.
4. There will be create folder logs in connection manager, but it's ok.

Reply:
the test is include sessionReadRemoteEmptyDirsRecursiveSync?because，It have error in my pc.

Reply:
Yes, test with sending a single file.

Reply:
> It have error in my pc

What's the error?

Reply:
registerReadEmptyDirsTask throw "Failed to registerReadEmptyDirsTask, already have same read job" in romote to local.

Reply:
throw “Failed to read dir, timeout”

Reply:
> throw "Failed to registerReadEmptyDirsTask, already have same read job"

Can you reproduce this?

> throw “Failed to read dir, timeout”

This happens when the controlled side is not built with this pr, or when network is bad or directory is very large.

Reply:
> This happens when the controlled side is not built with this pr.
可能是这个原因，我会更改报错的原因

Reply:
> throw "Failed to registerReadEmptyDirsTask, already have same read job"

Can you reproduce this?